README.md

```diff
@@ -22,34 +15,93 @@ To make this work, every application needs to do the following (at a minimum):
 
 ### ContainerPilot to the rescue!
 
-ContainerPilot is a helper written in Go to make it easier to containerize our applications. It can act as PID1 in the container and fork/exec the application. If the application exits then so does ContainerPilot. ContainerPilot reaps zombies, runs health checks, registers the app in the service catalog, watches the service catalog for changes, and runs your user-specified code at events in the lifecycle of the container to make it all work right.
-
-ContainerPilot is explicitly *not* a supervisor process. Although it can act as PID1 inside a container, it can't manage multiple services. And, if the shimmed process dies, so does ContainerPilot (and therefore the container itself). ContainerPilot will return the exit code of its shimmed process back to the Docker Engine or Triton, so that it appears as expected when you run `docker ps -a` and look for your exit codes. ContainerPilot also attaches stdout/stderr from your application to stdout/stderr of the container, so that `docker logs` works as expected.
-
-## Getting started
-
-See the [ContainerPilot documentation](https://www.joyent.com/containerpilot/docs) to get started, or jump to specific sections:
-
-- [Installation in a container](https://www.joyent.com/containerpilot/docs/installation)
-- [Configuration](https://www.joyent.com/containerpilot/docs/configuration)
-- [Service discovery](https://www.joyent.com/containerpilot/docs/service-discovery)
-- [Container lifecycle](https://www.joyent.com/containerpilot/docs/lifecycle)
-- [Health checks](https://www.joyent.com/containerpilot/docs/health)
-- [`preStart`, `preStop`, `postStop`](https://www.joyent.com/containerpilot/docs/start-stop)
-- [Signals and operations](https://www.joyent.com/containerpilot/docs/signals)
-- [Periodic tasks](https://www.joyent.com/containerpilot/docs/tasks)
-- [Telemetry](https://www.joyent.com/containerpilot/docs/telemetry)
-- [Coprocesses](https://www.joyent.com/containerpilot/docs/coprocesses)
-- [Frequently asked questions](https://www.joyent.com/containerpilot/docs/faq)
-- [Support](https://www.joyent.com/containerpilot/docs/support)
+ContainerPilot is an init system designed to live inside the container. It acts as a process supervisor, reaps zombies, run health checks, registers the app in the service catalog, watches the service catalog for changes, and runs your user-specified code at events in the lifecycle of the container to make it all work right. ContainerPilot uses Consul to coordinate global state among the application containers.
+
+## Quick Start Guide
+
+Check out our ["Hello, World" application](https://github.com/autopilotpattern/hello-world) on GitHub. Assuming you have Docker and Docker Compose available, it's as easy as:
```

Comment:
This could be an opportunity to link out to either our docs about Docker / Docker Compose on Triton (especially considering the recent changes with `triton-docker`.